vendor/engines/nucore_kfs/app/controllers/kfs_export_controller.rb

```diff
@@ -0,0 +1,7 @@
+class KfsExportController < ApplicationController
+  def show
+    @journal = Journal.find(params[:id])
+    @journal_rows = @journal.journal_rows
+    headers["Content-Disposition"] = "attachment;"
```

Comment:
We use `CSVHelper#set_csv_headers` in other places for setting the content disposition (and it allows you to specify a custom filename as well for the download).